pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/channel/ServiceUnitStateChannelImpl.java

```diff
@@ -298,6 +299,14 @@ public synchronized void start() throws PulsarServerException {
 
             ExtensibleLoadManagerImpl.createSystemTopic(pulsar, TOPIC);
 
+            if (config.isSystemTopicAndTopicLevelPoliciesEnabled()) {
```

Comment:
What happens if this flag returns false? Is that a valid way to configure Pulsar with this code? would it make sense to prevent such configuration in the first place, or at least log a warning?